packages/ndla-ui/src/locale/messages-nb.ts

```diff
@@ -490,6 +490,7 @@ const messages = {
     closeCompentenceGoalsFilter: 'Lukk filter',
     competenceGoalsNarrowBackButton: 'Tilbake',
     competenceGoalResourceSearchText: 'Vis ressurser til kompetansemål {{goal}}',
+    coreResourceSearchText: 'Vis ressurser til kjerneelement {{coreResourceText}}',
```

Comment:
Synes fremdeles det hadde gitt meinig å sende inn `{{code}}` for begge disse, men ikkje farlig.

Reply:
Aha, da skjønte jeg hva du mente, oppdatert!